Source/WebCore/platform/network/curl/CurlRequest.cpp

```diff
@@ -396,7 +396,14 @@ void CurlRequest::didReceiveHeaderFromMultipart(const Vector<String>& headers)
     for (auto header : headers)
```

Comment:
Wrtie `auto& header` not to copy a string.

Reply:
Fixed